prog/minio/setup_minio.rb

```diff
@@ -24,7 +24,7 @@ class Prog::Minio::SetupMinio < Prog::Base
     hosts = <<ECHO
 #{minio_server.cluster.generate_etc_hosts_entry}
 ECHO
-
+    minio_server.vm.sshable.cmd("sudo rm -f /etc/default/minio")
     minio_server.vm.sshable.cmd("sudo tee -a /etc/default/minio", stdin: def_minio)
     minio_server.vm.sshable.cmd("sudo tee -a /etc/hosts", stdin: hosts)
     minio_server.vm.sshable.cmd("sudo chown -R minio-user:minio-user /etc/default/minio")
```

Comment:
Should this be done as a rhizome program and using the atomic file stuff?

Reply:
Yes, I have refactored it properly